src/test/resources/com/owlcyberdefense/mil-std-2045/milstd2045.tdml

```diff
@@ -1314,6 +1324,9 @@ operator_reply_request_indicator set to 1">
         </ex:milstd2045_application_header>
       </tdml:dfdlInfoset>
     </infoset>
+    <validationErrors>
+      <error>Validation Error: cvc-type.3.1.3: The value 'Reserved_0' of element 'value_C' is not valid</error>
```

Comment:
Suggest make this string less specific so that the test is less fragile to very simple kinds of changes.

I would just put `<error>Reserved_0</error><error>value_C</error>`

Reply:
Updated